src/sentry/static/sentry/app/components/sidebar/index.jsx

```diff
@@ -0,0 +1,341 @@
+        <SidebarPanel title={t('My Bookmarks')}
+                      hidePanel={()=>this.hidePanel()}>
+          <IssueList
+            endpoint={`/organizations/${org.slug}/members/me/issues/bookmarked/`}
+            query={{
+              statsPeriod: '24h',
+              per_page: 10,
+              status: 'unresolved',
+            }}
+            pagination={false}
+            renderEmpty={() => <div className="sidebar-panel-empty" key="no">{t('You have no bookmarked issues.')}</div>}
+            ref="issueList"
+            showActions={false}
+            params={{orgId: org.slug}} />
+        </SidebarPanel>
+      }
+      {this.state.showPanel && this.state.currentPanel == 'history' &&
+        <SidebarPanel title={t('Recently Viewed')}
+                      hidePanel={()=>this.hidePanel()}>
+          <IssueList
+            endpoint={`/organizations/${org.slug}/members/me/issues/viewed/`}
+            query={{
+              statsPeriod: '24h',
+              per_page: 10,
+              status: 'unresolved',
+            }}
+            pagination={false}
+            renderEmpty={() => <div className="sidebar-panel-empty" key="none">{t('No recently viewed issues.')}</div>}
+            ref="issueList"
+            showActions={false}
+            params={{orgId: org.slug}} />
+        </SidebarPanel>
+      }
+      {this.state.showPanel && this.state.currentPanel == 'statusupdate' && status &&
+        <SidebarPanel title={t('Recent status updates')}
+                      hidePanel={()=>this.hidePanel()}>
+          <ul className="incident-list list-unstyled">
+            {status.incidents.map((incident) =>
+              <li className="incident-item" key={incident.id}>
+                <h4>{incident.title}</h4>
+                {incident.updates ?
+                  <div>
+                    <h6>Latest updates:</h6>
+                    <ul className="status-list list-unstyled">
+                      {incident.updates.map((update, key) =>
+                        <li className="status-item" key={key}>
+                          <p>{update}</p>
+                        </li>
+                      )}
+                    </ul>
+                  </div>
+                  :
+                  null
+                }
+                <p>
+                  <a href={incident.url} className="btn btn-default btn-sm">Learn more</a>
+                </p>
+              </li>
+            )}
+          </ul>
+        </SidebarPanel>
+      }
+    </div>);
+  },
+
+  renderRequiredActions() {
+    // TODO: investigate if this is seriously deprecated
+    let org = this.getOrganization();
+    let requiredAction = org && getFirstRequiredAdminAction(org);
+    let actionMessage;
+
+    if (org && requiredAction !== null) {
+      let slugId = requiredAction.ID.toLowerCase().replace(/_/g, '-');
+      let url = `/organizations/${org.slug}/actions/${slugId}/`;
+      actionMessage = (
```

Comment:
Changed.

Also – this is _probably_ deprecated code and I was going to  investigate removing afterwards.